src/charm.py

```diff
@@ -248,7 +256,17 @@ def _pgbouncer_layer(self) -> Layer:
             A pebble configuration layer for as many charm services as there are available CPU
             cores
         """
-        pebble_services = {}
+        pebble_services = {
+            "logrotate": {
+                "command": "logrotate -f /etc/logrotate.conf",
+                "startup": "enabled",
+                "backoff-delay": "24h",
+                "backoff-factor": 1,
```

Comment:
Poor man's cron daily. This will exec rotate on startup. If we want more granular behaviour, we should either add cron to the image or write a wrapper script.